scripts/start_integration_test_contracts_with_docker.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Get the latest Finschia tag
+TAG=$(curl "https://api.github.com/repos/Finschia/finschia/tags" | jq -r '.[0].name')
+
+# Copy init_single.sh from Finschia/finschia/init_single.sh
+init_single=$(mktemp)
+curl "https://raw.githubusercontent.com/Finschia/finschia/${TAG}/init_single.sh" -o $init_single
+
+# Remove v from tag prefix
+# ex. v1.0.0 -> 1.0.0
+TAG=$(echo "$TAG" | cut -c 2-)
+TEST_DOCKER_IMAGE=finschia/finschianode:${TAG}
+docker pull finschia/finschianode:${TAG}
+
+export FNSAD="docker run -i --rm -p 26656:26656 -p 26657:26657 -v ${HOME}/.finschia:/root/.finschia ${TEST_DOCKER_IMAGE} fnsad"
+
+# run Finschia/finschia/init_single.sh
+bash -xe $init_single
+
+container_id=$(docker run -d -p 26656:26656 -p 26657:26657 -v ${HOME}/.finschia:/root/.finschia ${TEST_DOCKER_IMAGE} fnsad start)
+
+docker exec ${container_id} apk add --no-cache jq bash curl && \
+docker cp ./integration_test_contracts.sh ${container_id}:/root/integration_test_contracts.sh && \
```

Comment:
why these are concated with `&&` instead of `;` or simply new line?

Reply:
“&&” was added before “integration_test_contracts.sh” because “integration_test_contracts.sh” requires success of the previous command.